packages/components/src/draggable/stories/style.css

```diff
@@ -0,0 +1,7 @@
+.draggable-storybook-wrapper-element .draggable-storybook-element {
+	background-color: #99faa4;
+}
+
+.draggable-storybook-element {
+	background-color: #fa99c8;
+}
```

Comment:
I wonder if it'd be better to just have a single `attachToOwnerDocument=true` story that demonstrates the z-index use case (instead of this CSS scoping demo), which is the exact use case that prompted you to implement this prop. Especially because with this CSS stuff, it's kind of impossible to understand what's going on without looking at the source code.

Reply:
That's a good question — I added the CSS example mostly to demonstrate that the original behaviour was retained (the `true` by default example). However, now that you've suggested going with a new prop that is false by default, there's less of a need for that.

I'll have a play around with `z-index` and see if I can consolidate on a single example that better reflects my use case. Thanks!